Source/Core/DiscIO/NANDContentLoader.cpp

```diff
@@ -495,6 +487,7 @@ u64 CNANDContentManager::Install_WiiWAD(const std::string& filename)
 
   tmd_file.WriteBytes(content_loader.GetTMDHeader(), CNANDContentLoader::TMD_HEADER_SIZE);
 
+  CSharedContent shared_content{Common::FromWhichRoot::FROM_CONFIGURED_ROOT};
```

Comment:
Whoops, I forgot that this was the _opposite_ problem; things could leak from/into the configured root as we were always using it instead of the session root in some cases (ES).

I'm not sure how we could construct it only when needed, apart from wrapping it inside a unique_ptr (and only constructing the unique_ptr on first use, and only once). As an aside, it looks like most WADs (system menus, IOSes, channels) have shared contents though, so we'll need to use this object in most cases.